src/button.test.visuals.ts

```diff
@@ -73,14 +76,15 @@ for (const story of stories.Button) {
               });
 
             await page.locator('glide-core-button').hover();
+            await page.mouse.down();
 
             await expect(page).toHaveScreenshot(
               `${test.titlePath.join('.')}.png`,
             );
           });
         });
 
-        test.describe('press', () => {
+        test.describe(':hover', () => {
```

Comment:
Weird diff. Just renamed `'press'` to `':active:'` and moved it above `':hover'`.